src/preferences/dialog/dlgprefbroadcast.cpp

```diff
@@ -129,6 +129,11 @@ DlgPrefBroadcast::~DlgPrefBroadcast() {
 }
 
 void DlgPrefBroadcast::slotResetToDefaults() {
+    UserSettingsPointer pConfig = m_pBroadcastSettings->getUserSettings();
+    QString defaultPath = QDir::currentPath();
```

Comment:
The current path will be most likely read only after Mixxx is installed. You should use the preferences path here.